website/src/user-guide/stub-files.md

```diff
@@ -12,6 +12,8 @@ Get inspired by [the stubs PHPStan itself uses](https://github.com/phpstan/phpst
 
 Stub files aren't a replacement for [discovering symbols](/user-guide/discovering-symbols) so if you're trying to fix errors like "Function not found" or "Class not found", check out the [discovering symbols](/user-guide/discovering-symbols) guide instead.
 
+PHPDoc within stub files will overwrite texisting PHPDoc. So makre sure to copy existing PHPDoc into stub files as well. See 
```

Comment:
typos: texisting -> existing, makre -> make 

also i think you forgot to paste a link due to the `See` at the end of the sentence

Reply:
I wanted to link to a place explaining the PHPDocs, but then removed the idea because there was no single helpful place.
Fixed the typos, thanks.